packages/mui-material/src/Tooltip/Tooltip.js

```diff
@@ -500,6 +500,25 @@ const Tooltip = React.forwardRef(function Tooltip(inProps, ref) {
     };
   }, [handleClose, open]);
 
+  React.useEffect(() => {
+    if (!open) {
+      return undefined;
+    }
+
+    /**
+     * @param {Event} nativeEvent
+     */
+    function handleScroll(nativeEvent) {
+      handleClose(nativeEvent);
+    }
+
+    document.addEventListener('scroll', handleScroll);
+
+    return () => {
+      document.removeEventListener('scroll', handleScroll);
+    };
```

Comment:
This feels a bit like a workaround. Why do the events that hide the tooltip don't trigger in the first place? https://github.com/mui/material-ui/issues/38379#issuecomment-1671097077